packages/ember-toucan-core/src/components/form/fields/checkbox-group.hbs

```diff
@@ -1,20 +1,41 @@
-<Form::Fieldset
-  @label={{@label}}
-  @hint={{@hint}}
-  @error={{@error}}
-  @isDisabled={{@isDisabled}}
-  aria-invalid={{if @error "true"}}
-  ...attributes
->
-  {{yield
-    (hash
-      CheckboxField=(component
-        (ensure-safe-component this.CheckboxFieldComponent)
-        name=@name
-        onChange=this.handleInput
-        isDisabled=@isDisabled
-        selectedValues=@value
-      )
-    )
-  }}
-</Form::Fieldset>
\ No newline at end of file
+<Form::Field as |field|>
+  <fieldset
+    aria-describedby="{{if @error field.errorId}} {{if @hint field.hintId}}"
+    disabled={{@isDisabled}}
+    {{! attribute aria-invalid is not supported by the element fieldset with the implicit role of group }}
```

Comment:
Ah - so no more `aria-invalid={{if @error "true"}}` due to this?

Reply:
Yeah I guess the linter (ember-template-lint?) doesn't catch this when we were adding it as `...attributes` on a component, but now that it can tell it's a group it doesn't want you to add `aria-invalid`?